### DISCUSSION TITLE
led.is_lit turns off the LED

### DISCUSSION THREAD

**Discussion by david-slatinek** [gpiozero/gpiozero#987] [Q&A] [answered]
Operating system: Manjaro 21.1.0 and Raspbian Buster
Python version: 3.7.3
Pi model: Raspberry Pi 4
GPIO Zero version: 1.6.2
      
I have a simple script to turn on the LED, called **led.py** which works as it is supposed to:
```python
#!/usr/bin/env python3

from gpiozero import LED
from signal import pause

pin = 18
led = LED(pin)

if __name__ == "__main__":
        led.on()
        pause()
```
I change the file permission with ```chmod u+x led.py``` and then I invoke the script by running ```./led.py &```

The problem arises when I run script **status.py**, which is supposed to print the LED status, (i.e on/off):
```python
#!/usr/bin/env python3

from gpiozero import LED
from signal import pause

pin = 18
led = LED(pin)

if __name__ == "__main__":
        print("LED status: " + ("ON" if led.is_lit else "OFF"))
```
However, the above script turns off the LED and prints that the LED is off. If I run ```jobs```, the **led.py** is still running in the background: ```[1]+  Running                 ./led.py &```

Is this a bug or am I missing something?

**Comment by bennuttall** [comment]
It depends how you've wired your LED. If it's wired GND -> GPIO then the code above is correct and `led.on()` should turn the LED on. If it's wired 3V3 -> GPIO then that's active-low and requires a change in construction i.e. `LED(18, active_high=False)`. See https://gpiozero.readthedocs.io/en/stable/api_output.html#led

**Reply by david-slatinek** [reply]
The wiring is correct and `led.on()` works. The problem is with `led.is_lit,` which turns the LED off.

**Reply by bennuttall** [reply]
You can't really run multiple scripts controlling the same pins. If you create a new LED object it will initialise it as off, so that'll be what's happening. You would be better sharing the logic about the state separately, like in a file.

**Reply by lurch** [reply]
In your `status.py` script I think you want to do `led = LED(pin, initial_value=None)`, as documented [here](https://gpiozero.readthedocs.io/en/stable/api_output.html#led).

**Comment by david-slatinek** [comment, marked as answer]
I have tried doing `led = LED(pin, initial_value=None)`, but it didn't work. I did, however, came with a solution proposed by @bennuttall.
> You can't really run multiple scripts controlling the same pins. If you create a new LED object it will initialise it as off, so that'll be what's happening. You would be better sharing the logic about the state separately, like in a file.

I am writing LED status to a file and then reading from it.
